charts/incubator/external-dns/templates/_args.tpl

```diff
@@ -0,0 +1,48 @@
+{{- define "externaldns.args" -}}
+args:
+  {{- with .Values.externaldns.provider }}
+  - --provider={{ . }}
+  {{- end -}}
+  {{- with .Values.externaldns.zoneidFilter }}
+  - --zone-id-filter={{ . }}
+  {{- end -}}
+  {{- with .Values.externaldns.cloudflareProxied }}
+  - --cloudflare-proxied={{ tpl . $ }}
+  {{- end }}
+  - --log-level={{ .Values.externaldns.logLevel }}
+  - --log-format={{ .Values.externaldns.logFormat }}
+  - --interval={{ .Values.externaldns.interval }}
+  {{- if .Values.externaldns.triggerLoopOnEvent }}
+  - --events
+  {{- end -}}
+  {{- range .Values.externaldns.sources }}
+  - --source={{ . }}
+  {{- end -}}
+  {{- with .Values.externaldns.policy }}
+  - --policy={{ . }}
+  {{- end -}}
+  {{- with .Values.externaldns.registry }}
+  - --registry={{ . }}
+  {{- end -}}
+  {{- with .Values.externaldns.txtOwnerId }}
+  - --txt-owner-id={{ . }}
+  {{- end -}}
+  {{- with .Values.externaldns.txtPrefix }}
+  - --txt-prefix={{ . }}
+  {{- end -}}
+  {{- if and (eq .Values.externaldns.txtPrefix "") (ne .Values.externaldns.txtSuffix "") }}
+  - --txt-suffix={{ .Values.externaldns.txtSuffix }}
+  {{- end -}}
+  {{- if .Values.externaldns.namespaced }}
+  - --namespace={{ include "tc.v1.common.lib.metadata.namespace" (dict "caller" "External-DNS" "rootCtx" $ "objectData" .Values) }}
+  {{- end -}}
+  {{- range .Values.externaldns.domainFilter }}
```

Comment:
This looks like it tries to loop over a list but on valeus and questions is a string